integration-tests/src/tests/hover.test.ts

```diff
@@ -73,15 +73,15 @@ suite('Bitbake Hover Test Suite', () => {
     await testHover(position, expected)
   }).timeout(BITBAKE_TIMEOUT)
 
-  test('Hover shows description for function defined into meta/classes-global/logging.bbclass', async () => {
+  test('Hover shows description for function defined into poky/meta/classes-global/logging.bbclass', async () => {
```

Comment:
I hit an intrusive virtual document change tooltip which we must avoid at all costs
![image](https://github.com/yoctoproject/vscode-bitbake/assets/19970889/ba4acb3a-f41c-4acd-81fe-98e9d2d407c4)

Reply:
didn't get it while testing today